src/CMakeLists.txt

```diff
@@ -32,23 +41,34 @@ execute_process(
 
 message(STATUS "Awkward includes : ${AWKWARD_INCLUDE}")
 message(STATUS "Awkward libraries: ${AWKWARD_LIBRARIES}")
-include_directories(BEFORE "${AWKWARD_INCLUDE}")
 
 find_library(CPU-KERNELS awkward-cpu-kernels REQUIRED HINTS ${AWKWARD_LIBRARIES})
 find_library(LIBAWKWARD awkward REQUIRED HINTS ${AWKWARD_LIBRARIES})
 find_library(LIBDL dl REQUIRED)
-message(STATUS "Libraries: ${CPU-KERNELS} ${LIBAWKWARD} ${LIBDL}")
+find_library(LIBARROW libarrow.so.400 REQUIRED HINTS ${PYARROW_LIBRARY_DIRS} NO_DEFAULT_PATH)
+find_library(LIBARROW_PYTHON libarrow_python.so.400 REQUIRED HINTS ${PYARROW_LIBRARY_DIRS})
```

Comment:
Ideally this shouldn't require the `.400` I'd think.

Reply:
I don't understand this at all.

Reply:
From the [find_library](https://cmake.org/cmake/help/latest/command/find_library.html) docs, it sounds like it was supposed to look up versioned SOs, but simply writing "arrow" and "arrow_python" didn't work, I think due to this versioning. Probably should try again.